lightning/src/ln/msgs.rs

```diff
@@ -49,6 +51,18 @@ use crate::ln::{PaymentPreimage, PaymentHash, PaymentSecret};
 /// 21 million * 10^8 * 1000
 pub(crate) const MAX_VALUE_MSAT: u64 = 21_000_000_0000_0000_000;
 
+/// A public nonce used for Musig2, comprised of two ECPoints
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct Musig2PublicNonce(pub(crate) PublicKey, pub(crate) PublicKey);
+
+/// A secret nonce used for Musig2, comprised of two SecretKeys
+#[derive(Clone, Debug, PartialEq)]
+pub struct Musig2SecretNonce(pub(crate) SecretKey, pub(crate) SecretKey);
+
+/// A partial signature that also contains the Musig2 nonce its signer used
```

Comment:
I think "channel-level signer" just not confused with a nested signer as even a partial signature could be generated by multiple "same-channel-side" signers. Or any notation making it clear which signer we're referring to.